<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>league_edit_menu</class>
 <widget class="QDialog" name="league_edit_menu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>League Editor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="page_title">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>League Editor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>6</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="list_title">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Teams in League</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="teams_list"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="buttons_to_edit_title">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Edit Teams in League:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_team_button">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Delete Team</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_team_button">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Edit Team</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="import_export_buttons_title">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Import/Export Teams:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="import_teams_button">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Import Teams</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="export_teams_button">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Export Teams</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="add_team_title">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add New Team to League:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="add_team_label">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Team Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="add_team_line_edit"/>
        </item>
        <item>
         <widget class="QPushButton" name="add_team_button">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Add Team</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>league_edit_menu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>league_edit_menu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
